guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: single
+title:  "Deploy Helm Charts"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.8" %}
+
+{% include toc %}
+
+Spinnaker surfaces a "Bake (Manifest)" stage to turn templates into manifests
+with the help of a templating engine. Currently, the only supported templating
+engine is [Helm](https://helm.sh/), by relying the `helm template` command. See
+more details [here](https://docs.helm.sh/helm/#helm_template).
+
+> Note: Make sure that you have configured [artifact support](/setup/artifacts)
+> in Spinnaker first. All Helm charts are fetched/stored as artifacts in
+> Spinnaker. Read more in the [reference pages](/reference/artifacts).
+
+## Configure the "Bake (Manifest)" stage
+
+When configuring the "Bake (Manifest)" stage, you can configure
+the following:
+
+* __The release name__ (required)
+
+  The Helm release name for this chart, and informs the name of the
+  artifact produced by this stage.
+
+* __The template artifact__ (required)
+
+  The Helm chart that you will be deploying, stored remotely as a
+  `.tar.gz` archive. You can produce this by running `helm package
+  /path/to/chart`. See more details
+  [here](https://docs.helm.sh/helm/#helm-package).
+
+* __Zero or more override artifacts__ (optional)
```

Comment:
Is "override artifacts" a term we use elsewhere? Otherwise perhaps "value files" or "value file artifacts", seeing as the Helm concept is a "Value File"?

Reply:
@lwander what say ye?